src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CollectionsShouldImplementGenericInterfaceTests.cs

```diff
@@ -15,37 +15,43 @@ namespace Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines.UnitTests
 {
     public class CollectionsShouldImplementGenericInterfaceTests
     {
-        private static DiagnosticResult GetCA1010CSharpResultAt(int line, int column, string typeName, string interfaceName, string genericInterfaceName)
-#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
I had to create the new overloads so I took a bit of extra and updated all other occurrences (that's one step forward for the no `RS0030`).